app/(afterLogin)/(deal)/deals/manage-deals/page.tsx

```
@@ -0,0 +1,122 @@
+import KanbanBoard from './_components/kanbanBoard';
+import DealSideBar from './_components/sideBar';
+import FilterModal from './_components/filter';
+import { LuSettings2 } from 'react-icons/lu';
+import FilterDropdown from '@/app/(afterLogin)/reports/_components/filterDropdown';
+
+const { Title, Text } = Typography;
+
+const ManageDealsPage: React.FC = () => {
+  const [searchValue, setSearchValue] = useState('');
+  const [isSideBarOpen, setIsSideBarOpen] = useState(false);
+  const [isFilterOpen, setIsFilterOpen] = useState(false);
```

ESLint

[error] 'isFilterOpen' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)